templates/artCard/index.html.twig

```diff
@@ -9,7 +9,7 @@
             {% for artCard in artCards %}
                 <div class="card artist-card col-6 col-md-4 d-flex flex-column  p-1 p-lg-2">
                     <img class="card-img img-fluid img-thumbnail bg-black"
-                         src="{{ artCard.pictureArt }}" alt="This is the profil picture of an artist"/>
+                         src="/uploads/artPicture/{{ artCard.pictureArt }}" alt="This is a picture of an art"/>
```

Comment:
No you can't have the directory hard coded in the twig

Reply:
Thats ok i use a twig global for call folder